packages/components/criteria/src/js/adder/criteria-adder.spec.js

```diff
@@ -124,6 +124,18 @@ describe('ouiCriteriaAdder', () => {
           title: 'contains',
         }]);
       });
+
+      it('should hide operators filed', () => {
+        const propertyMeta = find(mockData.properties, { name: 'example.string.noOperators' });
+        controller.columnModel = propertyMeta;
+        controller.onColumnChange();
+
+        // Must digest to apply ngHide
+        component.scope().$digest();
```

Comment:
Any reason why you use `$digest` instead of `$apply` ?

Reply:
We just need to refresh the current scope. But `$apply` will work in the same way here too :)